Sources/StreamChatUI/CommonViews/PlayPauseButton/PlayPauseButton.swift

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+open class PlayPauseButton: _Button, AppearanceProvider {
```

Comment:
Most of the logic here is the same as the one in PillButton. Would it make sense to have a common subclass that holds that logic?